src/auth-service/utils/control-access.js

```diff
@@ -360,6 +360,20 @@ const controlAccess = {
           let newResponse = Object.assign({}, responseFromListAccessToken);
           newResponse.message = "the token is valid";
           newResponse.data = newResponse.data[0];
+          try {
+            const service = request.headers
```

Comment:
for the new access tokens approach (`&token=TOKEN`), we are also able to log user activity from the token details. @BenjaminSsempala @NoahNsimbe @Codebmk @123MwanjeMike